megablocks/layers/arguments.py

```diff
@@ -24,7 +24,7 @@ class Arguments:
     moe_num_experts : int = 1
     moe_top_k : int = 1
     moe_capacity_factor : int = 1
-    moe_normalize_expert_weights: bool = False
+    moe_normalize_expert_weights: Union[int, float, bool] = False
```

Comment:
Rather than having `bool`, can we have this be `| None`? Then the logic using it is cleaner as well.

Reply:
updated this!